build-system/tasks/extension-helpers.js

```diff
@@ -486,9 +482,6 @@ async function buildExtension(
   }
 
   await compileJison(path.join(extDir, '**', '*.jison'));
-  if (name === 'amp-bind') {
-    await doBuildJs(jsBundles, 'ww.max.js', options);
-  }
```

Comment:
If this now happens automatically, I'd imagine some of the targets in this list will also happen automatically. If so, we may need to clean up this list to prevent them from being double-built. (Not 100% sure, but this is a strong hunch.)

https://github.com/ampproject/amphtml/blob/6bcd5338a29b25e6014fe463b63c5b3220fdf2b9/build-system/tasks/helpers.js#L204-L219

Reply:
Its more than likely the list can be completely empty :)

Reply:
Hmm. I actually may need to put `ww.max.js` back then. From the perspective of the lazy server, it isn't necessary, but for a regular build (i.e. `amp build --extensions=amp-bind`) it still is.

The error I was running into was the the lazy server will re-request `ww.max.js` even after amp-bind already did (causing a double watch). Will work on a different solution here.

Reply:
Gotcha. Not that it's also required by the lazy builder when you eager-build certain extensions. E.g. `amp --extensions=amp-bind`.

Reply:
True!